AmazonChimeSDKDemo/AmazonChimeSDKDemo/MeetingModel.swift

```diff
@@ -619,14 +619,10 @@ extension MeetingModel: VideoTileObserver {
             } else {
                 videoModel.addRemoteVideoTileState(tileState, completion: {
                     if self.activeMode == .video {
-                        // If the video is not currently being displayed, pause it
-                        if !self.videoModel.isRemoteVideoDisplaying(tileId: tileState.tileId) {
-                            self.currentMeetingSession.audioVideo.pauseRemoteVideoTile(tileId: tileState.tileId)
-                        }
                         self.videoModel.videoUpdatedHandler?()
```

Comment:
This should work, but more strict, when video tile added via subscription, there is no need to call resume(), right?

Reply:
Yeah, changed it to videoSubscriptionUpdatedHandeler() which uses subscription only.